### PR TITLE
TECH-294 groups page and nav updates

### PR DESCRIPTION
**Pull request by ericbroder**

### CODE REVIEW

File: pages/templates/pages/group_index_page.html [hunk 1]
```diff
@@ -69,6 +69,18 @@ <h4 class="mb20 mt0">No group in your area?</h4>
   </div>
 </div>
 
+<div class="container">
+  <div class="row">
+    <div class="col-xs-12">
+      <h3 class="mt40 mb20">
+        In addition to Local Groups, you can also take action on national
```
Comment (ericbroder) [inline pages/templates/pages/group_index_page.html]: What do you think about content here? Not sure what we want.

File: pages/templates/pages/group_index_page.html [hunk 2]
```diff
@@ -69,6 +69,18 @@ <h4 class="mb20 mt0">No group in your area?</h4>
   </div>
 </div>
 
+<div class="container pt4 pb2">
+  <div class="row">
+    <div class="col-xs-12">
+      <h3>
+        In addition to Local Groups, you can also take action on national
+        Our Revolution campaigns.
```
Comment (ericbroder) [inline pages/templates/pages/group_index_page.html]: @cjmabry thoughts about content here? design too?

File: pages/templates/partials/nav.html [hunk 3]
```diff
@@ -30,7 +30,14 @@
         <div class="row">
         <ul class="nav-list uppercase" id="main-nav-list">
           <li><a href="/about" class="{% if "/about" in request.path %}active{% endif %} about" id="about-nav-link">About</a></li>
-          <li><a href="/action" class="{% if "/action" in request.path %}active{% endif %} action" id="action-nav-link">Take Action</a></li>
+          {% with link_text="Local Groups" link_url="/groups/"  %}
+          <li>
+            <a href="{{ link_url }}" id="action-nav-link"
```
Comment (ericbroder) [inline pages/templates/partials/nav.html]: need to change id here

File: pages/templates/partials/nav.html [hunk 4]
```diff
@@ -30,7 +30,14 @@
...
+            class="{% if link_url in request.path %}active{% endif %} action">
```
Comment (ericbroder) [inline pages/templates/partials/nav.html]: need to change class too

File: pages/templates/pages/group_index_page.html (hunk 1 above)
Comment (ericbroder) [reply pages/templates/pages/group_index_page.html]: done

File: pages/templates/partials/nav.html (hunk 3 above)
Comment (ericbroder) [reply pages/templates/partials/nav.html]: done

File: pages/templates/partials/nav.html (hunk 4 above)
Comment (ericbroder) [reply pages/templates/partials/nav.html]: done

File: pages/templates/pages/group_index_page.html (hunk 2 above)
Comment (cjmabry) [reply pages/templates/pages/group_index_page.html]: This is fine for now. We'll redo when we do the public groups portal landing page redesign.

**Pull request merged**